jobs/sartregroup_NY_pythondatadev.html

```diff
@@ -0,0 +1,58 @@
+---
+title: Python Developer - HFT
+company: Sartre Group (Agency) on behalf of GTS.
+location: New York, US
+contract: Permanent
+contact:
+    name: Joe Jolly
+    email: joe.jolly@sartregroup.com
+    phone: +1 917 795 0584
+created: !!timestamp '2017-01-25'
+tags:
+  - python
+  - expert
+  - scientific libraries
+  - numpy
+  - pandas
+  - scipy
+  - numpy
+  - trading analytics
+  - HFT
+  - trade analysis
+  - data analysis
+  - quant finance
+---
+
+# **Overview**
+
+Python developer role, working in a leading financial institution (see company description).
```

Comment:
Perhaps you should say whether the role is aligned to the front-office or a trading team.